src/cc65/main.c

```diff
@@ -951,6 +962,9 @@ int main (int argc, char* argv[])
 
                 case 'O':
                     IS_Set (&Optimize, 1);
+                    IS_Set (&CodeSizeFactor, 100);
+                    IS_Set (&EnableRegVars, 0);
+                    IS_Set (&InlineStdFuncs, 0);
```

Comment:
I must admit that I don't understand why you added these three lines. I imagine that i.e.
```
-Or -O
```
should not turn register vars off, should it?